src/main/java/core/mate/academy/service/BuldozerProducer.java

```diff
@@ -0,0 +1,20 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BuldozerProducer implements MachineProducer {
+
+    @Override
+    public List<? extends Machine> get() {
+        List<Bulldozer> bulldozerList = new ArrayList<>();
```

Comment:
```suggestion
        List<Bulldozer> bulldozers = new ArrayList<>();
```
the same for other producers

Reply:
renamed